lib/rouge/lexers/ghc_core.rb

```diff
@@ -87,7 +87,7 @@ class GHCCore < RegexLexer
         rule %r/[A-Z][^\s.,(){}]*/, Keyword::Type
 
         # packages, e.g. 'ghc-prim-0.5.3:'
-        rule %r/(^[a-z].*?\d+\.\d+\.\d+)(:)(?=\S)/ do |m|
+        rule %r/([a-z].*?\d+\.\d+\.\d+)(:)(?=\S)/ do |m|
```

Comment:
I'm not sure why `^` was here in the first place - this form is used in the middle of lines in specs. But because of the `(` at the start of the regex, this never had an effect before.